pkg/controller/signup_test.go

```diff
@@ -481,7 +444,7 @@ func (s *TestSignupSuite) TestVerifyPhoneCodeHandler() {
 
 		require.Equal(s.T(), "Internal Server Error", bodyParams["status"])
 		require.InDelta(s.T(), float64(500), bodyParams["code"], 0.01)
-		require.Equal(s.T(), fmt.Sprintf("no user: error retrieving usersignup: %s", userID), bodyParams["message"])
+		require.Equal(s.T(), "no user: error retrieving usersignup: ", bodyParams["message"])
```

Comment:
A message ending with a ":" looks kinda unfinished. Where do we store the rest of it?

Reply:
good catch, just wait for the next PR :-) 
The thing is that I've modified the tests to rely on usernames only, but not the prod code yet. The prod code still tries to retrieve the UserSignup with username and userID, if there is some problem (like in this test case), the code returns an error with userID information only (which is empty in this case).
It's pretty hard to isolate some small incremental changes, but at the same time, the PR that would contain all related code would be simply huge.